auth/providers/azure/pop_tokenverifier_test.go

```diff
@@ -243,6 +268,10 @@ func TestPopTokenVerifier_Verify(t *testing.T) {
 	_, err := verifier.ValidatePopToken(validToken)
 	assert.NoError(t, err)
 
+	validToken, _ = GeneratePoPToken(time.Now().Unix(), "testHostname", tsClaimsTypeString)
```

Comment:
remind me what is that test compared to the previous one ?

Also no harm adding a description comment line on top of each test.

Reply:
Added comments for all tests